lms/static/sass/discussion/lms-discussion-bootstrap.scss

```diff
@@ -2228,6 +2232,11 @@ body.new-theme {
 // Layout
 .new-theme {
 	&.rtl {
+		.wmd-prompt-dialog {
+			#new-url-input {
+				direction: rtl !important;
```

Comment:
Why don't we use `direction: ltr !important;` here instead of `direction: ltr !important;` ?